Governing Board Public Minutes/2023-10-23.md

```diff
@@ -0,0 +1,688 @@
+   </td>
+   <td>X
+   </td>
+   <td>David A. Wheeler
+   </td>
+  </tr>
+  <tr>
+   <td>Director of Program Management
+   </td>
+   <td>X
+   </td>
+   <td>Amanda Martin
+   </td>
+  </tr>
+  <tr>
+   <td>Program Manager
+   </td>
+   <td>X
+   </td>
+   <td>Khahil White 
+   </td>
+  </tr>
+  <tr>
+   <td>Community Manager
+   </td>
+   <td>
+   </td>
+   <td>Cheuk Ho
+   </td>
+  </tr>
+  <tr>
+   <td>Sr. Marketing Manager
+   </td>
+   <td>X
+   </td>
+   <td>Jennifer Bly
+   </td>
+  </tr>
+  <tr>
+   <td>Inside Sales Representative & Manager
+   </td>
+   <td>X
+   </td>
+   <td>Randi Armour
+   </td>
+  </tr>
+  <tr>
+   <td>Executive Director, The Linux Foundation
+   </td>
+   <td>X
+   </td>
+   <td>Jim Zemlin
+   </td>
+  </tr>
+  <tr>
+   <td>SVP, GM of Projects, The Linux Foundation
+   </td>
+   <td>
+   </td>
+   <td>Mike Dolan
+   </td>
+  </tr>
+  <tr>
+   <td>SVP of Program Operations, The Linux Foundation
+   </td>
+   <td>X
+   </td>
+   <td>Todd Moore
+   </td>
+  </tr>
+</table>
+
+
+
+<table>
+  <tr>
+   <td><strong>Invited Guests</strong>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>Alpha Omega Rep
+   </td>
+   <td>X
+   </td>
+   <td>Bob Callaway
+   </td>
+  </tr>
+  <tr>
+   <td>Alpha Omega Rep
+   </td>
+   <td>X
+   </td>
+   <td>Micheal Scovetta
+   </td>
+  </tr>
+</table>
+
+## Call to Order 
+
+Omkhar Arasaratnam (OA) called the meeting to order at 9:04 am Pacific Time, Khahil White (KW), Dr. Amanda Martin (DM) and Adrianne Marcum (AM) recorded the minutes. Arun Gupta gave an introduction as the new Governing Board Chair. A quorum of Governing Board Members was established for the conduct of business, and the meeting, having been duly convened, was ready to proceed with business.
+
+## Attendance, Antitrust, Voting
+
+DM introduced the objectives and agenda for the meeting. There were no additional topics added.
+
+DM reminded the Governing Board of the Linux Foundation [antitrust policy](http://www.linuxfoundation.org/antitrust-policy) notice to which all meetings must adhere.
+
+Harry Toor (HT) noted two observing members invited to join the meeting. OA introduced new staff (Dana Wang, Chief Architect) who provided a brief introduction and insights on the OpenSSF landscape so far. 
+
+## Approval of Minutes
+
+* _WE HEREBY APPROVE: That the minutes of the August 17th, 2023 meeting of the Board of Directors, in the form attached hereto as Exhibit A, are hereby confirmed, approved and adopted. Minutes attached as [Exhibit A](https://drive.google.com/file/d/18vm0z2q7CShJm-z8C2FvNhR5y_KNemv-/view?usp=sharing)._
+    * Stephen Augustus motioned to approve.
+    * Rao seconded the motion. 
+    * All in favor.
+
+## Approval of MVSR
+
+* _WE HEREBY APPROVE: That the Mission, Vision, Values, and Strategy as defined in the attached in [Exhibit B](https://docs.google.com/document/d/1_oLsODuHRv2kYYn8meBUc9T6e68BZQFsknYBG642acw/edit#heading=h.e9uhthj01hdc) is approved._
```

Comment:
I'm not able to access the document here, should it be publcily viewable?

Reply:
They are now publicly on openssf.org. This is the doc they board viewed, and they have access to the doc, so for historical purposes, we keep the link here.